react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -69,6 +69,12 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
         }
     }
 
+    @objc
+    func nativeLoggingHook(_ message: String, logLevel: Int) {
+        guard let logLevel = LogLevel.init(rawValue: logLevel) else { return }
```

Comment:
(optional)syntactic sugar: we can drop .init and use LogLevel(...)

Reply:
Thx! 🙇 51183cf